### PR TITLE
Feature8nj

### PR DESCRIPTION
**Pull request by MadhurangaWije**

### CODE REVIEW

File: src/db/main.bal [hunk 1]
```diff
@@ -6,7 +6,13 @@ public function main() returns error?{
 
     check dbClient.close();
     // some changes 2
-   
+    // some more and more chages
+   // changed added by feature3
+   // more changes 2
+   // more changes
```
Comment (MadhurangaWije) [inline src/db/main.bal:12]: ghjfyuuh

File: src/db/main.bal [hunk 2]
```diff
@@ -6,7 +6,13 @@ public function main() returns error?{
 
     check dbClient.close();
     // some changes 2
-   
+    // some more and more chages
+   // changed added by feature3
+   // more changes 2
```
Comment (MadhurangaWije) [inline src/db/main.bal:11]: ssss

File: src/db/main.bal [hunk 3]
```diff
@@ -6,7 +6,13 @@ public function main() returns error?{
...
+   // more changes
+   // more and more changes
+   // more changes by the feature 7
```
Comment (MadhurangaWije) [inline src/db/main.bal:14]: sss

File: src/db/main.bal [hunk 4]
```diff
@@ -6,7 +6,13 @@ public function main() returns error?{
...
+   // more changed by the featrure 8
 }
```
Comment (MadhurangaWije) [inline src/db/main.bal:17]: ytf

File: src/db/main.bal [hunk 5]
```diff
@@ -6,7 +6,13 @@ public function main() returns error?{
 
     check dbClient.close();
     // some changes 2
-   
+    // some more and more chages
+   // changed added by feature3
```
Comment (MadhurangaWije) [inline src/db/main.bal:10]: h

Comment (MadhurangaWije) [reply src/db/main.bal:10]: ### h

File: src/db/main.bal [hunk 6]
```diff
@@ -6,7 +6,13 @@ public function main() returns error?{
 
     check dbClient.close();
     // some changes 2
-   
+    // some more and more chages
```
Comment (MadhurangaWije) [inline src/db/main.bal:9]: hh

File: src/db/main.bal [hunk 7]
```diff
@@ -6,7 +6,13 @@ public function main() returns error?{
 
     check dbClient.close();
```
Comment (Pubuduboteju) [inline src/db/main.bal:7]: ABC

**Review by Pubuduboteju — CHANGES_REQUESTED** [review]

**Pull request closed without merging**